### PR TITLE
Docs: Custom Version Classes

### PR DESCRIPTION
**Pull request by jaredbeck**
Separate "Advantages" from "Configuration"

@batter let me know if you want to review docs changes like this or if I should just push to master.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -563,7 +563,13 @@ class Post < ActiveRecord::Base
 end
 ```
 
-This allows you to store each model's versions in a separate table, which is useful if you have a lot of versions being created.
+### Advantages
+
+1. For models which have a lot of versions, storing each model's versions in a
+   separate table can improve the performance of certain database queries.
+1. Store different version [metadata](#storing-metadata) for different models.
```
Comment (batter) [inline README.md]: `2.`?

Comment (jaredbeck) [inline README.md]: Heh, that's a markdown thing. :D  You don't have to increment your numbers, md will do it for you.  Using all ones makes long lists easy to reorder.

Comment (batter) [inline README.md]: Ahh, good thinking

**Comment by batter** [comment]
+1

**Comment by batter** [comment]
> @batter let me know if you want to review docs changes like this or if I should just push to master.

I think it's good to get a 2nd set of eyes for some stuff, but for error fixes and minor improvements to documentation, not a huge deal either way. So for documentation updates I would say feel free to push to master unless you want some review.

**Comment by jaredbeck** [comment]
> I think it's good to get a 2nd set of eyes for some stuff, but for error fixes and minor improvements to documentation, not a huge deal either way. So for documentation updates I would say feel free to push to master unless you want some review.

Roger wilco.

**Pull request merged**